src/handlers/getCurrentUser.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express-async-router'
+import * as db from '../gateways/storage'
+
+export async function handler(req: express.Request, res: express.Response) {
+  const user = await db.getUser(req.user?.d_id)
+  if (user) {
+    res.status(200).send(user)
```

Comment:
🚨🚨 We cannot do this!

As it stands, `db.getUser` return the whole row for a user, including its `discordAccessToken` column. The `discordAccessToken` should NEVER be exposed by the API, as it would allow anyone who captured that data to make calls to the discord API using that token, pretending to have the authorization from that user. 

Either `db.getUser` needs to select a few specific columns we want to expose, or we need to filter them out before sending the response.